stdlib/src/main/scala/chisel3/std/fifo/CombFifo.scala

```diff
@@ -0,0 +1,19 @@
+// Author: Martin Schoeberl (martin@jopdesign.com)
+// License: this code is released into the public domain, see README.md and http://unlicense.org/
+
+package chisel3.std.fifo
+
+import chisel3._
+
+/**
+  * A FIFO queue combining the memory FIFO with a single stage double buffer FIFO
```

Comment:
the scaladoc in these is a little bit mis-formatted (suprised scalafmt is not flagging it?). I think we should have a really high bar for documentation for anything in the stdlib